sesdev/__init__.py

```diff
@@ -610,18 +610,25 @@ def _gen_settings_dict(
     for folder in synced_folder:
         try:
             src, dst = folder.split(':')
-            if not all([path.isabs(x) for x in [src, dst]]):
-                raise OptionValueError('--synced-folder',
-                                       "Please provide absolute paths for "
-                                       "synced folder paths",
-                                       folder)
-            if not path.exists(src):
-                raise OptionValueError('--synced-folder',
-                                       "Path to the source synced folder must exist",
-                                       src)
-
+            if path.isabs(src) and path.isabs(dst):
+                pass
+            else:
+                raise OptionValueError(
+                    "--synced-folder",
+                    "Please provide absolute paths for synced folder paths",
+                    folder
+                    )
+            if path.exists(src):
+                pass
+            else:
+                raise OptionValueError(
+                    "--synced-folder",
+                    "Path to the source synced folder must exist",
+                    src
+                    )
```

Comment:
Just gave it a second thought and removing the square brackets makes it a generator already:

```suggestion
            if not all(path.isabs(x) for x in [src, dst]):
                raise OptionValueError('--synced-folder',
                                       "Please provide absolute paths for "
                                       "synced folder paths",
                                       folder)
            if not path.exists(src):
                raise OptionValueError('--synced-folder',
                                       "Path to the source synced folder must exist",
                                       src)
```

To be frank, I'd prefer it more concise, but it works in both cases.

Reply:
I read somewhere that once Python programmers learn how to use comprehensions, they tend to use them everywhere. Maybe there was a **little** bit of that going on here...